File: internal/stream/telemetry.go
package stream

type Telemetry struct {
	FPS      float64 `json:"fps"`
	Score    float32 `json:"score"`
	Lost     bool    `json:"lost"`
	Tracking bool    `json:"tracking"`
	PosX     int     `json:"posX"`
	PosY     int     `json:"posY"`
}

File: frontend/src/types/telemetry.ts
export type TrackerState = "idle" | "tracking" | "lost";

export type Telemetry = {
  fps: number;
  score: number;
  state: TrackerState;
  trackingOn: boolean;
  lost: boolean;
  posX: number | null;
  posY: number | null;
};
